dashboard/app/views/pd/workshop_mailer/organizer_should_close_reminder.html.haml

```diff
@@ -10,7 +10,7 @@
 
 %p
   Your
-  = link_to "#{@workshop.course} workshop", "/pd/workshop_dashboard/workshops/#{@workshop.id}"
+  = link_to "#{@workshop.course} workshop", CDO.studio_url("/pd/workshop_dashboard/workshops/#{@workshop.id}"), CDO.default_scheme
```

Comment:
Same here. 

Check out the [mail previews](https://github.com/code-dot-org/code-dot-org/blob/e9c55af9f77219d16fa7cda95ed684ac6477c353/dashboard/test/mailers/previews/pd_workshop_mailer_preview.rb) locally and make sure the links render properly. Note they won't navigate to a valid page because the ids are from dummy data in the preview, but you can verify the url format.